src/jit/codegenlinear.cpp

```diff
@@ -710,11 +710,7 @@ void CodeGen::genSpillVar(GenTreePtr tree)
     bool needsSpill = ((tree->gtFlags & GTF_VAR_DEF) == 0 && varDsc->lvIsInReg());
     if (needsSpill)
     {
-        var_types lclTyp = varDsc->TypeGet();
-        if (varDsc->lvNormalizeOnStore())
-        {
-            lclTyp = genActualType(lclTyp);
-        }
+        var_types lclTyp = genActualType(varDsc->TypeGet());
```

Comment:
>  var_types lclTyp = genActualType(varDsc->TypeGet()); [](start = 6, length = 54)

It would be a good idea to add a comment explaining why we use actual type here.